packages/server-core/src/networking/instance-provision/instance-provision.class.ts

```diff
@@ -394,7 +394,7 @@ export async function checkForDuplicatedAssignments({
       setTimeout(() => {
         retry = false
         resolve(false)
-      }, config.server.instanceserverUnreachableTimeoutSeconds * 1000) // timeout after 2 seconds
+      }, config.server['instance-server'].UnreachableTimeoutSeconds * 1000) // timeout after 2 seconds
```

Comment:
This should be reverted back. The unreachable timeout seconds setting is still in server-settings.

Reply:
Fixed it myself